app/Controllers/feedController.php

```diff
@@ -547,6 +547,7 @@ public function actualizeAction() {
 		$url = Minz_Request::param('url');
 		$force = Minz_Request::param('force');
 		$maxFeeds = (int)Minz_Request::param('maxFeeds');
+		$noredirect = Minz_Request::paramBoolean('noredirect');
```

Comment:
There is only one use of that variable. Might be useless to set it.
You could use the parameter content directly where you need it.